src/python/pants/engine/isolated_process.py

```diff
@@ -29,7 +29,7 @@ def _run_command(binary, sandbox_dir, process_request):
                            stderr=subprocess.PIPE,
                            stdout=subprocess.PIPE,
                            cwd=sandbox_dir)
-  # TODO At some point, we may want to replace this blocking wait with a timed one that returns
```

Comment:
This entire method should now be unused and deleted; if it isn't, it would mean some invoke had not been ported to the new codepaths.

Reply:
some of the tests  (in `test_isolated_process.py`) have input/output file requirements that still use this method. I left this in, so that I wouldn't have to remove those tests. Once I have an equivalent set of tests (exercising input/output files) using the process execution in Rust, I will remove this and the corresponding tests.